hashin.py

```diff
@@ -741,6 +751,11 @@ def get_parser():
         action="store_true",
         default=False,
     )
+    parser.add_argument(
+        "--index-url",
+        help="package index url (default https://pypi.org/)",
+        default="https://pypi.org/",
```

Comment:
How about moving this to the top of the file. Something like this:

```python
DEFAULT_INDEX_URL = os.environ.get('INDEX_URL', 'https://pypi.org')
```

and then, here, do...:

```python
parser.add_argument(
        "--index-url",
        help="package index url (default {0})".format(DEFAULT_INDEX_URL),
        default=DEFAULT_INDEX_URL,
```